gc_run.go

```diff
@@ -0,0 +1,37 @@
+//go:build !windows
+
+package main
+
+import (
+	"context"
+	"syscall"
+)
+
+func (nd *Node) periodicGC(ctx context.Context, threshold float64) error {
+	var stat syscall.Statfs_t
+
+	err := syscall.Statfs(nd.dataDir, &stat)
```

Comment:
IIUC this is deprecated https://pkg.go.dev/syscall, but there's an alternative with the x/syscall package (even if we don't use the OS-agnostic one) https://stackoverflow.com/a/20110856

Reply:
It is not marked as deprecated?